frontend/src/lib/FileUploadClient.ts

```diff
@@ -18,47 +18,116 @@
 import { CancelToken } from "axios"
 import HttpClient from "lib/HttpClient"
 import { SessionInfo } from "lib/SessionInfo"
+import _ from "lodash"
```

Comment:
`import { isEqual } from "lodash"`? 
It may help during tree shaking. Though I'm not sure!